src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -77,21 +77,33 @@ public static bool IsAttributeNameContext(this SyntaxTree syntaxTree, int positi
 
             var tokenOnLeftOfPosition = syntaxTree.FindTokenOnLeftOfPosition(position, cancellationToken);
             var token = tokenOnLeftOfPosition.GetPreviousTokenIfTouchingWord(position);
+            var parent = token.Parent;
 
             var modifierTokens = syntaxTree.GetPrecedingModifiers(position, tokenOnLeftOfPosition);
             if (modifierTokens.IsEmpty())
             {
-                return false;
+                if (token.IsKind(SyntaxKind.CloseBracketToken)
+                    && parent.IsKind(SyntaxKind.AttributeList, out AttributeListSyntax attributeList)
+                    && !(attributeList.Target is { Identifier: { RawKind: (int)SyntaxKind.AssemblyKeyword } }))
```

Comment:
what about `[module: ` attributes?

Reply:
Not sure. Those aren't handled by `ISyntaxFacts.IsGloablAttribute` either.

Reply:
not really getting the relevance.  i guess i'm asking.  if the user writes:

```c#
[module: Whatever]
$$
```

does completion work properly?

Reply:
It wouldn't, but now it does.